be/src/util/block_compression.cpp

```diff
@@ -73,6 +73,13 @@ class Lz4BlockCompression : public BlockCompressionCodec {
     }
 
     Status compress(const Slice& input, faststring* output) override {
+        if (input.size > INT_MAX) {
```

Comment:
unlikely ？

Reply:
For the compression of a block, this judgment is very lightweight, I don't think it is necessary to use `unlikely`

Reply:
maybe we only need DCHECK here?